### PR TITLE
[WIP] Highlight reminder events on conrad show

### LINKED ISSUE #46: Highlight events on conrad show for which reminders have been set
Using colorama.

### PR DESCRIPTION
**Pull request by IamShivamJaiswal**
Fixes #46.

### CODE REVIEW

File: conrad/cli.py [hunk 1]
```diff
@@ -171,6 +171,12 @@ def _show(ctx, *args, **kwargs):
     events = list(
         session.query(Event).filter(*filters).order_by(Event.start_date).all()
     )
+    reminders = list(
```
Comment (vinayak-mehta) [inline conrad/cli.py]: Instead of creating a list here, you can query the reminders table in the `for event in events` loop.

File: conrad/cli.py [hunk 2]
```diff
@@ -185,18 +191,48 @@ def _show(ctx, *args, **kwargs):
...
-                    event.city,
-                    event.state,
-                    event.country,
-                    event.start_date.strftime("%Y-%m-%d"),
-                    event.end_date.strftime("%Y-%m-%d"),
-                ]
-            )
+            if(reminders):
+                for reminder, __ in reminders:
+                    if(reminder.id==event.id):
```
Comment (vinayak-mehta) [inline conrad/cli.py]: Please run `black` on your code.

File: conrad/cli.py [hunk 3]
```diff
@@ -185,18 +191,48 @@ def _show(ctx, *args, **kwargs):
...
+                        events_output.append(
+                        [
+                            Fore.GREEN + Style.BRIGHT + event.id + Style.RESET_ALL,
```
Comment (vinayak-mehta) [inline conrad/cli.py]: Just highlighting the id with "yellow" should be good.

Comment (IamShivamJaiswal) [reply conrad/cli.py]: updated

File: conrad/cli.py (hunk 2 above)
Comment (IamShivamJaiswal) [reply conrad/cli.py]: updated

File: conrad/cli.py (hunk 1 above)
Comment (IamShivamJaiswal) [reply conrad/cli.py]: updated

**Comment by IamShivamJaiswal** [comment]
Now when everything is correct but I don't know why it's not highlighting text, If you please check and let me know

**Pull request closed without merging**